### ISSUE TITLE
Active support requirement is old

### ISSUE THREAD

**Issue by jondruse** [bluepill-rb/bluepill#15]
Is there an actual problem with using activesupport > 4?  Maybe I can help.  Thanks.

**Comment by sgonyea** [comment]
I was coming here to actually post on this issue.  Bluepill shouldn't have a dependency on ActiveSupport.   If there's anything that we need from that library, we should copy and paste it into an internal module.  Making ActiveSupport a requirement leads to some bizarre conflicts.

**Comment by jondruse** [comment]
If that's true, that would be awesome.  We're actually looking for an alternative to bluepill because of this, but would obviously love to keep using it.

**Comment by darrenleeweber** [comment]
Ran into a crash related to this dependency, i.e. see https://github.com/rails/rails/issues/14664; the traceback that triggers it is:
```
bundle stderr: /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/number_helper.rb:3:in `<module:NumberHelper>': uninitialized constant ActiveSupport::Autoload (NameError)
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/number_helper.rb:2:in `<module:ActiveSupport>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/number_helper.rb:1:in `<top (required)>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/core_ext/numeric/conversions.rb:2:in `require'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/core_ext/numeric/conversions.rb:2:in `<top (required)>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/core_ext/numeric.rb:3:in `require'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/activesupport-4.2.5/lib/active_support/core_ext/numeric.rb:3:in `<top (required)>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/bluepill-0.0.68/lib/bluepill.rb:14:in `require'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/bluepill-0.0.68/lib/bluepill.rb:14:in `<top (required)>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/bluepill-0.0.68/bin/bluepill:3:in `require'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/gems/bluepill-0.0.68/bin/bluepill:3:in `<top (required)>'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/bin/bluepill:23:in `load'
	from /var/sdr2service/sdr-preservation-core/shared/bundle/ruby/2.1.0/bin/bluepill:23:in `<main>'
```

**Comment by darrenleeweber** [comment]
Attempted fix in https://github.com/bluepill-rb/bluepill/pull/42

**Comment by akzhan** [comment]
Just released 0.1.3 with fix
